node/openrpc.go

```diff
@@ -142,10 +140,15 @@ func newOpenRPCDocument() *go_openrpc_reflect.Document {
 		// the mock subscription receiver type RPCSubscription.
 		// This pattern matches all strings that start with Subscribe and are suffixed with a non-zero
 		// number of A-z characters.
-		if isPubSub(method.Type) {
-			// This catches MOST of them (except SubscribeSyncStatus)
-			return false
+		pkgStr := method.Type.In(0).String()
+		if (isPubSub(method.Type) && pkgStr == "*eth.PublicEthereumAPI") || pkgStr == "*filters.PublicFilterAPI" {
```

Comment:
This is the important part.

I'm using the stringer condition because using `reflect.TypeOf(&eth.PublicEthereumAPI)` will cause an import cycle error.

I _think_ its correct that all the methods from `*filters.PublicFilterAPI` can be skipped since, AFAIU, these are all "unconventional" methods in service of either `eth` or `debug` or `trace` methods which use subs.

Reply:
Related to https://github.com/etclabscore/core-geth/pull/356/files#r618692596